dlt-database/entity/0001-init_db/Account.ts

```diff
@@ -0,0 +1,80 @@
+import {
+  Entity,
+  PrimaryGeneratedColumn,
+  Column,
+  CreateDateColumn,
+  ManyToOne,
+  JoinColumn,
+  OneToOne,
+  OneToMany,
+  ManyToMany,
+  JoinTable,
+} from 'typeorm'
+import { User } from './User'
+import { Community } from './Community'
+import { TransactionRecipe } from './TransactionRecipe'
+import { ConfirmedTransaction } from './ConfirmedTransaction'
+import { DecimalTransformer } from '../../src/typeorm/DecimalTransformer'
+import Decimal from 'decimal.js-light'
+
+@Entity('accounts')
+export class Account {
+  @PrimaryGeneratedColumn('increment', { unsigned: true })
+  id: number
+
+  @ManyToOne(() => User, (user) => user.accounts) // Assuming you have a User entity with 'accounts' relation
+  @JoinColumn({ name: 'user_id' })
+  user: User
+
+  // if user id is null, account belongs to community gmw or auf
+  @Column({ name: 'user_id', type: 'int', unsigned: true, nullable: true })
+  userId?: number
+
+  @Column({ name: 'derivation_index', type: 'int', unsigned: true })
+  derivationIndex: number
+
+  @Column({ name: 'derive2_pubkey', type: 'binary', length: 32, unique: true })
+  derive2Pubkey: Buffer
+
+  @Column({ type: 'tinyint', unsigned: true })
+  type: number
+
+  @CreateDateColumn({
+    name: 'created_at',
+    type: 'datetime',
+    default: () => 'CURRENT_TIMESTAMP',
```

Comment:
i recognize the dimension of a date column we have to take care.
we want to treat date columns with 3 digits for milliseconds. 

Then you have to define in the migration a datetime column with datetime(3) and for default setting like CURRENT_TIMESTAMP you have to add the ms too, like CURRENT_TIMESTAMP(3)

Please care for this on all datetime columns.

Reply:
done

Reply:
Here in the entity also necessary
to set the dimension of ms at the default setting CURRENT_TIMESTAMP(3)

Reply:
I hope this time I really have them all.